tests/test_operation.py

```diff
@@ -709,6 +709,23 @@ def test_operation_multiply_invalid(self):
             T = X @ Z
             Y @ T
 
+    def test_eigvals(self):
+        """Test that the correct eigenvalues are returned for the Tensor"""
+        X = qml.PauliX(0)
+        Y = qml.PauliY(2)
+        t = Tensor(X, Y)
+        assert np.array_equal(t.eigvals, np.kron(qml.PauliX.eigvals, qml.PauliY.eigvals))
```

Comment:
actually a great test! I wouldn't think of doing it this way

Reply:
Would you have had another way of testing in mind?

Reply:
If it was me, I probably would have compared against the numeric values, but I like this better!